shared/actions/fs/platform-specific.desktop.js

```diff
@@ -230,34 +230,62 @@ const driverEnableFuse = (state, action) =>
           .then(() => FsGen.createRefreshDriverStatus())
   )
 
-function* uninstallKBFSConfirm() {
-  const resp = yield Saga.callUntyped(
-    () =>
-      new Promise((resolve, reject) =>
-        SafeElectron.getDialog().showMessageBox(
-          null,
-          {
-            buttons: ['Remove & Restart', 'Cancel'],
-            detail: `Are you sure you want to remove Keybase from ${fileUIName} and restart the app?`,
-            message: `Remove Keybase from ${fileUIName}`,
-            type: 'question',
-          },
-          resp => resolve(resp)
-        )
-      )
+const uninstallKBFSConfirmPromise = () =>
```

Comment:
Super nit: just `uninstallKBFSConfirm`, same below. We name other action handlers this way too.